packages/apps/plugins/plugin-help/src/components/Shortcuts/ShortcutsHints.tsx

```diff
@@ -22,7 +21,7 @@ export const ShortcutsHints = ({ onClose }: { onClose: () => void }) => {
 
   // TODO(burdon): Factor out.
   // TODO(burdon): How to access all translations across plugins?
-  const toString = (label: Label) => (Array.isArray(label) ? t(...label) : label);
+  const toString = (label: any) => (Array.isArray(label) ? t(label[0], label[1]) : label);
```

Comment:
Can this be something other than `any`? Otherwise, maybe comment on this, or make it a guard of some kind.